docs/deployment/customization/portal.properties-Reference.md

```diff
@@ -31,6 +31,22 @@ the `db.connection_string` instead.
 db.tomcat_resource_name=jdbc/cbioportal
 ```
 
+### Python-specific connection URL
+
+``` 
+db.connection_string_python= 
+
+``` 
+
+Setting for python-specific connection URL in order to connect using both python and java at the entrypoint.
+
+The format of the `db.connection_string_python` is: 
+
+``` 
+jdbc:mysql://localhost:3306/cbiodb
+
+``` 
```

Comment:
```suggestion
The format of the `db.connection_string_python` is: 

``` 
jdbc:mysql://localhost:3306/cbiodb

``` 
```

The format of the `db.connection_string_python` is identical to `db.connection_string`.